base/setup/usetup/usetup.c

```diff
@@ -4461,8 +4476,23 @@ BootLoaderHarddiskMbrPage(PINPUT_RECORD Ir)
                                    PartitionType);
     if (!NT_SUCCESS(Status))
     {
-        MUIDisplayError(ERROR_WRITE_BOOT, Ir, POPUP_WAIT_ENTER);
-        return QUIT_PAGE;
+        /* Check the partition type, if we have FAT on it */
+        if (PartitionType == PARTITION_FAT32 ||
```

Comment:
I would change this check to check `PARTITION_LINUX`, and just have an 'else' case with no condition for fat.
this makes it impossible to continue (as it was before)